aspnetcore/security/authentication/social/google-logins.md

```diff
@@ -9,13 +9,13 @@ uid: security/authentication/google-logins
 ---
 # Google external login setup in ASP.NET Core
 
-By [Valeriy Novytskyy](https://github.com/01binary) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+By [Valeriy Novytskyy](https://github.com/01binary), [Rick Anderson](https://twitter.com/RickAndMSFT) and [Sharaf Abacery](https://github.com/sharafabacery)
 
 This tutorial shows you how to enable users to sign in with their Google account using the ASP.NET Core project created on the [previous page](xref:security/authentication/social/index).
 
 ## Create the Google OAuth 2.0 Client ID and secret
 
-* Follow the guidance in [Integrating Google Sign-In into your web app](https://developers.google.com/identity/sign-in/web/sign-in) (Google documentation).
+* Follow the guidance in [Integrating Google Sign-In into your web app](https://developers.google.com/identity/gsi/web/guides/overview) (Google documentation)
```

Comment:
that goes to:
 Google Identity Services is migrating to FedCM APIs. Follow the [migration guide](https://developers.google.com/identity/gsi/web/guides/fedcm-migration) to review potential changes and avoid negative impacts for user sign-in to your website.

Shouldn't we point to *** FedCM APIs*** ???

Reply:
ok,I see your point ,be more  general for need to sign-in